src/test/java/org/openstreetmap/atlas/geography/atlas/change/validators/ChangeAtlasEdgeValidatorTest.java

```diff
@@ -45,12 +45,13 @@ public void testModifyEdgeAndNode()
         changeBuilder.add(getFeatureChangeMovedNode());
 
         final Change change = changeBuilder.get();
-        Assert.assertEquals("[Edge: id=39001000001, startNode=38999000000, endNode=39002000000, "
-                + "polyLine=LINESTRING (-122.2457961 37.592796, -122.2450237 37.5926929, "
-                + "-122.2441049 37.5930666, -122.2429584 37.5926993), "
-                + "[Tags: [last_edit_user_name => myself], [last_edit_changeset => 1], "
-                + "[last_edit_time => 1513719782000], [last_edit_user_id => 1], [name => primary], "
-                + "[highway => primary], [last_edit_version => 1]]]",
+        Assert.assertEquals(
+                "[Edge: id=39001000001, startNode=38999000000, endNode=39002000000, "
```

Comment:
Does that pass spotless?

Reply:
I think it does. I ran `spotlessApply` before committing.